app/src/main/java/com/nextcloud/talk/diagnose/DiagnoseContentComposable.kt

```diff
@@ -37,13 +65,41 @@ fun DiagnoseContentComposable(data: State<List<DiagnoseActivity.DiagnoseElement>
     ) {
         data.value.forEach { element ->
             when (element) {
-                is DiagnoseActivity.DiagnoseElement.DiagnoseHeadline -> Text(
-                    modifier = Modifier.padding(vertical = 16.dp),
-                    text = element.headline,
-                    color = MaterialTheme.colorScheme.primary,
-                    fontSize = LocalDensity.current.run { dimensionResource(R.dimen.headline_text_size).toPx().toSp() },
-                    fontWeight = FontWeight.Bold
-                )
+                is DiagnoseActivity.DiagnoseElement.DiagnoseHeadline -> {
+                    if (element.headline == "Test push notifications") {
+                        Button(
+                            modifier = Modifier
+                                .wrapContentSize()
+                                .padding(vertical = 8.dp),
+                            onClick = {
+                                diagnoseViewModel.fetchTestPushResult()
+                            }
+                        ) {
+                            Text(
+                                text = element.headline,
+                                modifier = Modifier
+                                    .fillMaxWidth()
+                                    .padding(vertical = 8.dp),
+                                color = colorResource(R.color.high_emphasis_text),
```

Comment:
Text color must not be set here. With R.color.high_emphasis_text the color would not work well together with theming